patches/qt6/0001-Use-absolute-URLs-for-submodules.patch

```diff
@@ -1,401 +0,0 @@
-From ca8d35f53741f2043b3a75a597858696e6f07d6c Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Kacper=20S=C5=82omi=C5=84ski?=
- <kacper.slominski72@gmail.com>
-Date: Wed, 6 Nov 2024 12:28:37 +0100
-Subject: [PATCH] Use absolute URLs for submodules
-
----
- .gitmodules | 100 ++++++++++++++++++++++++++--------------------------
- 1 file changed, 50 insertions(+), 50 deletions(-)
-
-diff --git a/.gitmodules b/.gitmodules
-index ee888bb..c3e6b44 100644
---- a/.gitmodules
-+++ b/.gitmodules
-@@ -6,47 +6,47 @@
- [submodule "qtsvg"]
- 	depends = qtbase
- 	path = qtsvg
--	url = ../qtsvg.git
-+	url = https://code.qt.io/qt/qtsvg.git
- 	branch = 6.6.2
- 	status = addon
```

Comment:
Why is this patch dropped? The relative URLs are a problem on xbbs (where we pull from our mirror and not from upstream).

Reply:
Because the QT superbuild system was changed and this change breaks the entire build. The superbuild will now deduce the base url part from the parent repo (the one we clone from the recipe) and will then add the url part to it, or so it seems. Surprisingly, this only happens for a submodule of qtdeclarative? I fought this for 2 hours and then gave up. The correct fix is to get us into upstream so we don't have to use patched repo's, the workaround in the meantime is to not pull this from the mirror (or mirror everything and hope for the best).